dev/com.ibm.ws.com.unboundid/src/com/ibm/ws/com/unboundid/InMemoryTDSLDAPServer.java

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package com.ibm.ws.com.unboundid;
+
+import com.ibm.websphere.simplicity.log.Log;
+
+/**
+ * An in memory UnboundID LDAP server designed to contain the same data as the Tivoli Directory Server instance of the ApacheDS stand-alone LDAP Servers.
+ *
+ * This InMemoryTDSLDAPServer is currently using a null schema to host the data, this could be replaced in the future with a robust activeDirectory schema design
```

Comment:
```suggestion
 * This InMemoryTDSLDAPServer is currently using a null schema to host the data, this could be replaced in the future with a robust TDS schema design
```